vendor/brave-ios/Ledger/Data/migrate.sql

```diff
@@ -1,6 +1,7 @@
 -- Create Tables
 
 CREATE TABLE activity_info(publisher_id LONGVARCHAR NOT NULL,duration INTEGER DEFAULT 0 NOT NULL,visits INTEGER DEFAULT 0 NOT NULL,score DOUBLE DEFAULT 0 NOT NULL,percent INTEGER DEFAULT 0 NOT NULL,weight DOUBLE DEFAULT 0 NOT NULL,reconcile_stamp INTEGER DEFAULT 0 NOT NULL,CONSTRAINT activity_unique UNIQUE (publisher_id, reconcile_stamp) CONSTRAINT fk_activity_info_publisher_id    FOREIGN KEY (publisher_id)    REFERENCES publisher_info (publisher_id)    ON DELETE CASCADE);
+CREATE TABLE balance_report_info(balance_report_id LONGVARCHAR PRIMARY KEY NOT NULL UNIQUE,grants DOUBLE DEFAULT 0 NOT NULL,earning_from_ads DOUBLE DEFAULT 0 NOT NULL,auto_contribute DOUBLE DEFAULT 0 NOT NULL,recurring_donation DOUBLE DEFAULT 0 NOT NULL,one_time_donation DOUBLE DEFAULT 0 NOT NULL);
```

Comment:
changes in this file are not needed as we only migrate ios to version 10 and then shared lib takes over